about.html

```diff
@@ -268,6 +230,38 @@ <h3 class="mbr-section-title mbr-fonts-style align-center mb-4 display-2">
     </div>
 </section>
 
+<section data-bs-version="5.1" class="team2 cid-ter6x3oElh" xmlns="http://www.w3.org/1999/html" id="team2-1h">
+    
+
+    
+    
+    <div class="container">
+        <div class="card">
+            <div class="card-wrapper">
+                <div class="row align-items-center">
+                    <div class="col-12 col-md-4">
+                        <div class="image-wrapper">
+                            <img src="assets/images/picture-day-585x556.webp" alt="Mobirise Website Builder">
+                        </div>
+                    </div>
+                    <div class="col-12 col-md">
+                        <div class="card-box">
+                            <h5 class="card-title mbr-fonts-style m-0 mb-3 display-5">
+                                <strong>Probablyry</strong></h5>
```

Comment:
Double check probablyry is still active - showing last activity of 7/17, which is beyond our 1-month activity window.